indexer/packages/postgres/src/db/migrations/migration_files/20241002144813_change_affiliate_info_fee_columns.ts

```diff
@@ -0,0 +1,23 @@
+import * as Knex from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  return knex
+    .schema
+    .alterTable('affiliate_info', (table) => {
+      table.dropColumn('totalReferredFees');
+      table.dropColumn('referredNetProtocolEarnings');
+      table.decimal('totalReferredTakerFees', null).notNullable().defaultTo(0);
+      table.decimal('totalReferredMakerFees', null).notNullable().defaultTo(0);
```

Comment:
Can we separate positive maker fees from negative maker fees?

Reply:
Im reluctant to do so directly in the table because it introduces another layer of if/else to the columns. I think if users of this table wish to differentiate, they can do so in their query. What do you think?

Reply:
I will add totalReferredMakerRebates